src/store/modules/accounts/hdWallet.js

```diff
@@ -21,8 +21,10 @@ export default {
     nextAccountIdx: 1,
   },
   actions: {
-    async isAccountUsed({ rootGetters }, address) {
-      return rootGetters['sdkPlugin/sdk'].api.getAccountByPubkey(address).then(() => true, () => false);
+    async isAccountUsed({ rootGetters: { store } }, address) {
```

Comment:
rootGetters are not needed here
```suggestion
    async isAccountUsed(context, address) {
```

Reply:
Totally true. I think this was a mistake as the store is not accessible from the root getters.

Reply:
Thanks.